tests/travis/checkout_test_against_branch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPT_DIR=`dirname $0`
+
+if [ "$TEST_AGAINST_PIWIK_BRANCH" == "" ]; then
+    if [ "$TEST_AGAINST_CORE" == "latest_stable" ]; then # test against the latest stable release of piwik core (including betas & release candidates)
```

Comment:
> I don't really want to rename it since it will break existing builds that may use it... I guess it's ok if we're the only ones using though.

+1, let's remove completely  `latest_stable` for now (we don't want to use it anymore)

and then let's make all plugins use `minimum_required_piwik` instead. sound ok?